examples/ruby-on-rails/Earthfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:2.6.5
+WORKDIR /ruby-on-rails-example
+ENV BUNDLER_VERSION=2.1.4
+
+deps:
+    RUN curl -sL https://deb.nodesource.com/setup_12.x  | bash - && \
+      apt-get install -y nodejs
+    RUN gem install bundler:2.1.4
+    COPY Gemfile Gemfile.lock ./
+    RUN bundle config build.nokogiri --use-system-libraries
+    RUN bundle check || bundle install
+    SAVE ARTIFACT Gemfile.lock AS LOCAL ./Gemfile.lock
+    SAVE IMAGE
+build:
+    FROM +deps
+    COPY . .
+    SAVE ARTIFACT . /.
+    SAVE IMAGE
+docker:
+    FROM +build
+    EXPOSE 3000
+    CMD ["bundle", "exec", "rails", "server", "-b", "0.0.0.0"]
+    SAVE IMAGE ruby-on-rails-example:latest
+migrate:
+    FROM +build
+    RUN bundle exec rails db:migrate
+test:
+    FROM +build
+    RUN bundle exec rails test
+console:
+    FROM +build
+    RUN bundle exec rails console
```

Comment:
Possibly here too - though we might be abusing the pattern.

`RUN --push` ensures that the command is not cached and will run it every time.

Reply:
Actually this one is probably not a good idea since it's not interacting with an external service.

Reply:
Do you mean to remove `--push`? @vladaionescu

Reply:
Yes - it should only be used for db:migrate, I think. But @agbell might know the Ruby patterns better.